### PR TITLE
Feedback

### PR DESCRIPTION
**Pull request by OverSx**
:wave:! GitHub Classroom created this pull request as a place for your teacher to leave feedback on your work. It will update automatically. **Don’t close or merge this pull request**, unless you’re instructed to do so by your teacher.

In this pull request, your teacher can leave comments and feedback on your code. Click the **Subscribe** button to be notified if that happens.

Click the **Files changed** or **Commits** tab to see all of the changes pushed to `master` since the assignment started. Your teacher can see this too.

<details>
<summary>
<strong>Notes for teachers</strong>
</summary>

Use this PR to leave feedback. Here are some tips:

- Click the **Files changed** tab to see all of the changes pushed to `master` since the assignment started. To leave comments on specific lines of code, put your cursor over a line of code and click the blue **+** (plus sign). To learn more about comments, read “[Commenting on a pull request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request)”.
- Click the **Commits** tab to see the commits pushed to `master`. Click a commit to see specific changes.
- If you turned on autograding, then click the **Checks** tab to see the results.
- This page is an overview. It shows commits, line comments, and general comments. You can leave a general comment below.

For more information about this pull request, read “[Leaving assignment feedback in GitHub](https://classroom.github.com/help/leaving-feedback-in-github)”.
</details>

Subscribed: @Bd0ptimus

### CODE REVIEW

File: DZ1_Bui-The-Zung_UTS-22/DZ1/Program.cs [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+            Area = calArea(x, y);
+            Console.WriteLine("Square = {0} \n", Math.Round(Area, 3));
+
+            x = null;
+            y = null;
+            GC.Collect();
+
+            //Pause the console app in 1 minute
+            Console.WriteLine("Pausing 1 Minute");
+            System.Threading.Thread.Sleep(60000);
```
Comment (OverSx) [inline DZ1_Bui-The-Zung_UTS-22/DZ1/Program.cs:139]: Вместо паузы в 1 минуту, достаточно использовать Console.Readkey()

**Comment by OverSx** [comment]
Добавьте проверку существования треугольника, координаты которого задает пользователь.

**Comment by Bd0ptimus** [comment]
> Добавьте проверку существования треугольника, координаты которого задает пользователь.
Я добавил возможность проверять треугольник. Пожалуйста, проверьте еще раз

**Comment by d-makarenko** [comment]
Оценка: 4/4

Прошу в следующий раз не переусложнять задачу. Чем проще, тем лучше

**Pull request merged**